dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/OneAgentMetadataEnricher.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2020 Dynatrace LLC
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.dynatrace.opentelemetry.metric;
+
+import com.google.common.base.Strings;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class OneAgentMetadataEnricher {
```

Comment:
```suggestion
final class OneAgentMetadataEnricher {
```
No need to expose this. Same for `MetricAdapter`.